src/styles/annotator/highlights.scss

```diff
@@ -204,12 +204,14 @@
 .hypothesis-highlights-always-on .hypothesis-shape-highlight {
   // This color is similar to the default highlight fill, but darker so it has
   // more contrast when used as a border.
-  border-color: #edd72b;
+  --highlight-color: #edd72b;
+  border-color: var(--highlight-color);
+  background-color: color-mix(in srgb, var(--highlight-color) 30%, transparent);
```

Comment:
[`color-mix`](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-mix#browser_compatibility) is ~3 years old in Safari, ~2 in Firefox and Chrome. In older browsers, the highlight have a border only.